src/Microsoft.PowerShell.Security/security/CredentialCommands.cs

```diff
@@ -86,20 +100,18 @@ public GetCredentialCommand() : base()
         /// </summary>
         protected override void BeginProcessing()
         {
-            if (!string.IsNullOrEmpty(this.Message))
-            {
-                string caption = UtilsStrings.PromptForCredential_DefaultCaption;
-                string message = this.Message;
+            string caption = this.Title ?? UtilsStrings.PromptForCredential_DefaultCaption;
+            string message = this.Message ?? "Enter your credentials.";
```

Comment:
Ah yeah true. so just:
string message = this.Message ?? null;
?
I dont like that the message is mandatory, as if one just want to specifi username or the title.
where is no reason for message to be mandatory?